README.md

```diff
@@ -22,79 +30,165 @@
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> **[Local Storage]**
+  > A feature that allows you to store data on the client-side
+  **[Modules]**
+  > Modules can help keep code organized, improve maintainability, and reduce the likelihood of naming conflicts.
+  **[CRUD]**
+  > These are the basic operations used in databases and applications for managing data.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
 
-## Built With
+- Live Demo is not available
 
-- HTML
-- CSS
-- JS
-- Webpack
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Key Features
-- [Setting up of Linters]
-- [Setting up of webpack]
-- [Adding Index.html and style.css]
-- [styling a text using class giving a appropriate font size,margin,image,padding]
 
-## Live Demo
+## 💻 Getting Started <a name="getting-started"></a>
 
-[Live Demo]()
+To get a local copy up and running, follow these steps.
 
-## Getting started
+### Prerequisites
 
-Go to the Folder where want to copy this repo and type the following cmds in you terminal:
+In order to run this project you need:
 
-- git clone git@github.com:HenaryKhuraijam/To-do-List.git
+- A text editor (preferably Visual Studio Code)
+- Github
 
-## Prerequisites
+### Setup
 
-- Knowledge HTML, CSS, JS and Webpack
-- VSCode or other text editor installed. [Link to download VSCode](https://code.visualstudio.com/download)
-- Node package. [Link to download node](https://nodejs.org/en/download/)
+Clone this repository to your desired folder:
 
-## To Install type the following command
+cd my-folder
+  git clone git@github.com:HenaryKhuraijam/To-do-List.git
 
-- npm install
 
-## To Run App type the following command in your terminal
+### Install
 
-  - npm start
+Install this project with:
 
-### To Run test cases type the following command
+  -  [Git](https://git-scm.com/downloads)
+  -  [Node](https://nodejs.org/en/download/)
 
-  - npm test
+### Usage
 
-## Author
+To run the project, execute the following command:
+- After Cloning this repo to your local machine
+- Open the `index.html` in your browser
+- Open with Live Server
 
-👤 Khuraijam Henary Singh
+### Run tests
+To track linter errors locally follow these steps:  
+
+Download all the dependencies run:
+```
+npm install
+```
+Track HTML linter errors run:
+```
+npx hint .
+```
+Track CSS linter errors run:
+```
+npx stylelint "**/*.{css,scss}"
+```
+Track JavaScript linter errors run:
+```
+npx eslint .
+```
+### Deployment
+You can deploy this project using:
+GitHub pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Khuraijam Henary Singh**
 
 - GitHub: [@HenaryKhuraijam](https://github.com/HenaryKhuraijam)
 - Twitter: [@HenaryKhuraijam](https://twitter.com/HenaryKhuraijam)
 - LinkedIn: [Henary Khuraijam](https://www.linkedin.com/in/henary-khuraijam)
 
+👤 **Samuel**
```

Comment:
For professional repositories, please let's avoid broken links and ensure that all links in the README file are working.

Please kindly fix the broken link in the `Authors Section` under 👤 **Khuraijam Henary Singh**. 🛠👍

👉 `LinkedIn: [Henary Khuraijam](https://www.linkedin.com/in/henary-khuraijam)`

![image](https://user-images.githubusercontent.com/78251927/231722649-952ba53a-f91e-47f6-ba19-9b3fb45ba65b.png)